src/radical/pilot/agent/scheduler/base.py

```diff
@@ -892,61 +907,70 @@ def _schedule_incoming(self):
 
```

Comment:
> `self.slot_status("before schedule incoming [%d]" % len(to_schedule))`

`priority` should be considered here - sum of lengths of pools per priority

Reply:
thanks, done.